parsing/printast.ml

```diff
@@ -37,8 +37,9 @@ let fmt_location f loc =
 
 let rec fmt_longident_aux f x =
   match x with
-  | Longident.Lident (s) -> fprintf f "%s" s
-  | Longident.Ldot (y, s) -> fprintf f "%a.%s" fmt_longident_aux y s
+  | Longident.Lident { txt = s; _ } -> fprintf f "%s" s
+  | Longident.Ldot (y, { txt = s; _ }) ->
+      fprintf f "%a.%s" fmt_longident_aux y s
```

Comment:
For the ast printer, we probably want to print the new location information.

Reply:
I started thinking about it and I am wondering if it's really a good thing to do. There is a lot of new locations and it might clutter the output a lot.